source/index.html

```diff
@@ -241,7 +241,9 @@ <h2 class="visually-hidden">Наши контакты</h2>
         </ul>
       </div>
       <div class="footer__copyright copyright">
-        <a class="copyright__link" href="https://htmlacademy.ru/intensive/adaptive">Разработано</a>
+        <a class="copyright__link" href="https://htmlacademy.ru/intensive/adaptive">Разработано
+          <span class="visually-hidden">HTML Academy.</span>
+        </a>
```

Comment:
Лучше не писать одну часть содержимого в строке, а другую на новой строке.
Когда код читаешь, если тег закрывается на отдельной строке, обычно не ищешь после открытого тега еще содержимое. Поэтому лучше так (например):
```suggestion
        <a class="copyright__link" href="https://htmlacademy.ru/intensive/adaptive">
          Разработано
          <span class="visually-hidden">HTML Academy.</span>
        </a>
```

Reply:
стоит ли поправить это во всем документе, чтобы было единообразие? (в смысле чтобы всегда текст внутри параграфов, спанов, кнопок и т.п. был с новой строки)

Reply:
Нет, не обязательно.
Вообще, я для себя лично придумал правило:
> Если внутри тега есть другой тег, то его содержимое должно быть на новой строке

Но это мое правило, а не твое =) Ты можешь другими принципами руководствоваться.
Единственное, чего бы не хотелось — это комбинации, когда часть содержимого в строке, а часть на новой